File: Desafios em C#/2927 - Imprevistos Natalinos.cs
using System; 

class URI {

    static void Main(string[] args) { 
        
        string[] linha = Console.ReadLine().Split(' ');

        int numAlunos = int.Parse(linha[0]);
        int numPc = int.Parse(linha[1]);
        int qtsPcsQuebrados = int.Parse(linha[2]);
        int qtsPcsSemComp = int.Parse(linha[3]);
        
        if (numPc - (numAlunos + qtsPcsQuebrados + qtsPcsSemComp + 1) >= 0){
          Console.WriteLine("Igor feliz!");
        } 
        else {
          if ((qtsPcsSemComp / 2) < qtsPcsQuebrados) Console.WriteLine("Caio, a culpa eh sua!");
          else Console.WriteLine("Igor bolado!");
        }

    }

}

File: Desafios em JavaScript/2927 - Imprevistos Natalinos.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

var numAlunos = parseInt(lines.shift());
var numPc = parseInt(lines.shift());
var qtsPcsQuebrados = parseInt(lines.shift());
var qtsPcsSemComp = parseInt(lines.shift());


if (numPc - (numAlunos + qtsPcsQuebrados + qtsPcsSemComp + 1) >= 0){
    console.log("Igor feliz!");
} 
else {
    if ((qtsPcsSemComp / 2) < qtsPcsQuebrados) console.log("Caio, a culpa eh sua!");
    else console.log("Igor bolado!");
}